pyproject.toml

```diff
@@ -4,3 +4,22 @@ requires = ["Cython>=0.27.1", "setuptools", "wheel"]
 [tool.black]
 line-length = 100
 target-version = ['py36', 'py37', 'py38', 'py39']
+
+[tool.cibuildwheel]
+# Numpy and Scipy began moving to manylinux2014 for 64-bit builds in late 2021,
+# and are generally dropping support for 32-bit architectures entirely.  Our
+# setup is a bit of a compromise; we provide longer support for Linux-i686 where
+# Numpy and Scipy have dropped it completely, but follow their lead on 64-bit.
+manylinux-x86_64-image = "manylinux2014"
```

Comment:
As we discussed offline I'm wondering if it makes more sense to split this out for terra 0.20.0 as a general thing. We'll need this for 3.10 regardless I think, but for python 3.7-3.9 we should try to keep with manylinux2010 if possible. We can probably get away with this by doing a `before-test` to install a manylinux2010 version of scipy and then override it for 3.10.